src/katgpucbf/xbgpu/engine.py

```diff
@@ -794,9 +967,14 @@ def __init__(
         # time. This keeps the pipelines synchronised to avoid running out of
         # RxQueueItems.
         self._active_in_sem = asyncio.BoundedSemaphore(1)
-        self._pipelines = [
-            XPipeline(output, self, context, tx_enabled) for output in outputs if isinstance(output, XOutput)
-        ]
+
+        self._pipelines: list[Pipeline]
+        x_outputs = [output for output in outputs if isinstance(output, XOutput)]
+        b_outputs = [output for output in outputs if isinstance(output, BOutput)]
+        self._pipelines = [XPipeline(output, self, context, tx_enabled) for output in x_outputs]
+        if len(b_outputs) == 1:
+            # TODO: Update once more BOutput's are supported
+            self._pipelines.append(BPipeline(b_outputs[0], self, context, tx_enabled))
```

Comment:
Looks like if you specify multiple b_outputs, it will silently ignore all of them?